test/e2e/notifier/update/update.go

```diff
@@ -194,3 +196,59 @@ func (c *context) testSKipUpdateEvent(t *testing.T) {
 	// Resetting original configuration as per test_config
 	utils.AllowedEventKindsMap[utils.EventKind{Resource: "v1/pods", Namespace: "all", EventType: "update"}] = true
 }
+
+func (c *context) testSkipWrongSetting(t *testing.T) {
```

Comment:
@A-kanksh-a Can we move this test case to `testSKipUpdateEvent` function?

Reply:
@PrasadG193  the step here is different,` testSKipUpdateEvent` function checks for value in `AllowedEventKindsMap` while `testSkipWrongSetting` function creates and update the resource and then checks on the Diff calculated on the old and new objects, since the UpdateSeting field is invalid the diff calculated would be blank.